src/Show.jl

```diff
@@ -49,6 +49,26 @@ function irrchartypes(t::Table)
 	return t.table.original_m
 end
 
+"""
+    irrchars(t::Table)
```

Comment:
I would just call this `nrchars` as well.

And `irrchartypes` should be nirrchartypes` (or rather in OSCAR style: `n_irrchartypes` -- or even `number_of_irreducible_character_types` or whatever... )